app/assets/stylesheets/_colors.sass

```diff
@@ -0,0 +1,7 @@
+// This is the RGSoC teams-app color stack:
+
+$main: 		#333
+$railsred:	#d82222
+$lightgray:	#ddd
+$darkgray: 	#999
+$white: 	#fff
```

Comment:
AAAAAH I keep on changing my indentation setting but they don't seem to stick. googling.it.now.